pkg/kapp/resources/resource_filter.go

```diff
@@ -92,6 +94,24 @@ func (f ResourceFilter) Matches(resource Resource) bool {
 		}
 	}
 
+	if len(f.Labels) > 0 {
+		var matched bool = false
+		for _, label := range f.Labels {
+			labelSelector, err := labels.Parse(label)
+			if err != nil {
+				return false
```

Comment:
i wonder if we should panic here for now, since otherwise we are swallowing error and skipping other matchers. eventually we could add Validate() may be to ResourceFilter.